### PR TITLE
[Flutter] update add segmentation to view

### PR DESCRIPTION
**Pull request by arifBurakDemiray**

### CODE REVIEW

File: flutter/next.md [hunk 1]
```diff
@@ -572,20 +572,32 @@ final String? anotherViewID = Countly.instance.views.startView("HomePage", segme
...
   Here is an example for how to add segmentation to a view using its ID:
 </p>
-<pre><code class="dart">Map&lt;String, Object&gt; segmentation = {
-  "Cats": 123,
-  "Moons": 9.98,
-  "Moose": "Deer"
+<pre><code class="dart">String viewID = '';
+Countly.instance.views.startView('HomePage').then((value) = {
+  if(value != null){
+    viewID = value
```
Comment (peterBrxwn) [inline flutter/next.md:591]: add semi colon at the end of this line

File: flutter/next.md [hunk 2]
```diff
@@ -572,20 +572,32 @@ final String? anotherViewID = Countly.instance.views.startView("HomePage", segme
...
-  "Cats": 123,
-  "Moons": 9.98,
-  "Moose": "Deer"
+<pre><code class="dart">String viewName = 'HomePage';
+Countly.instance.views.startView(viewName);
+
+Map&lt;String, Object&gt; segmentation = {
+  'Cats': 123,
+  'Moons': 9.98,
+  'Moose': 'Deer'
```
Comment (peterBrxwn) [inline flutter/next.md:581]: trailing comma

File: flutter/next.md [hunk 3]
```diff
@@ -572,20 +572,32 @@ final String? anotherViewID = Countly.instance.views.startView("HomePage", segme
...
 <p>
   Here is an example for how to add segmentation to a view using its ID:
 </p>
-<pre><code class="dart">Map&lt;String, Object&gt; segmentation = {
-  "Cats": 123,
-  "Moons": 9.98,
-  "Moose": "Deer"
+<pre><code class="dart">String viewID = '';
+Countly.instance.views.startView('HomePage').then((value) = {
+  if(value != null){
```
Comment (peterBrxwn) [inline flutter/next.md:590]: formatting. Add space before "{"

**Review by peterBrxwn — APPROVED** [review]

File: flutter/next.md (hunk 2 above)
Comment (arifBurakDemiray) [reply flutter/next.md:581]: what happened to it

Comment (arifBurakDemiray) [reply flutter/next.md:581]: got it

**Pull request merged**